lib/util/middleware.js

```diff
@@ -139,3 +140,12 @@ export async function deleteTrip (middlewareConfig, token, id) {
     }
   }
 }
+
+export async function checkItinerary (middlewareConfig, token, monitoredTrip) {
```

Comment:
jsdoc needed. Also, this is another case where there is a lot of logic in the component state that I would prefer that we move to actions and persist in the reducer if possible.

Reply:
This function is now an action in https://github.com/opentripplanner/otp-react-redux/pull/242/commits/ea97ce635402164e163c1f46be35c6ac7d971697.